core/storage/suggestion/gae_models_test.py

```diff
@@ -318,6 +318,49 @@ def test_query_suggestions_by_language(self):
             len(suggestion_models.GeneralSuggestionModel.query_suggestions(
                 queries)), 1)
 
+    def test_get_translation_suggestions_in_review_by_exp_id_happy_case(self):
+        suggestion_models.GeneralSuggestionModel.create(
+            suggestion_models.SUGGESTION_TYPE_TRANSLATE_CONTENT,
+            suggestion_models.TARGET_TYPE_EXPLORATION,
+            'exp1', self.target_version_at_submission,
+            suggestion_models.STATUS_IN_REVIEW, 'author_3',
+            'reviewer_2', self.change_cmd, self.score_category,
+            'exploration.exp1.thread_6', self.translation_language_code)
+        suggestion_models.GeneralSuggestionModel.create(
+            suggestion_models.SUGGESTION_TYPE_TRANSLATE_CONTENT,
+            suggestion_models.TARGET_TYPE_EXPLORATION,
+            'exp1', self.target_version_at_submission,
+            suggestion_models.STATUS_IN_REVIEW, 'author_4',
+            'reviewer_2', self.change_cmd, self.score_category,
+            'exploration.exp1.thread_7', self.translation_language_code)
+
+        suggestions = (
+            suggestion_models.GeneralSuggestionModel
+            .get_translation_suggestions_in_review_by_exp_id('exp1'))
+
+        self.assertEqual(len(suggestions), 2)
+        self.assertEqual(suggestions[0].target_id, 'exp1')
+        self.assertEqual(
+            suggestions[0].suggestion_type,
+            suggestion_models.SUGGESTION_TYPE_TRANSLATE_CONTENT)
+        self.assertEqual(
+            suggestions[0].status,
+            suggestion_models.STATUS_IN_REVIEW)
```

Comment:
Suggest also adding tests for:
- Retrieving non-translation suggestions in review (shouldn't fetch)
- Retrieving translation suggestions not in review (shouldn't fetch)
- Retrieving more matches that DEFAULT_QUERY_LIMIT (should truncate--also, suggest overriding DEFAULT_QUERY_LIMIT with a swap to simplify that test case)
- Retrieving no matches for a valid exploration when there are no suggestions present

Reply:
Done!

> Retrieving no matches for a valid exploration when there are no suggestions present

The function/layer doesn't depend on the existence of the target entity i.e, exploration and it seems out of context (i.e, we haven't created exploration models in the suggestion model's unit test anywhere in this file) for the suggestion_model.get_translation_suggestions_in_review_by_exp_i. [The test would be the same as checking there isn't ant suggestion with target_id X.] I'm fine adding one here, I'll import exp_services and create exploration and then make the call. Let me know if you think we should add one! :)